### PR TITLE
Update readme before archiving

### PR DESCRIPTION
**Pull request by ricardozanini**
Updating the README file before we archive this project. See: https://github.com/kiegroup/kogito-serverless-operator/pull/101

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,151 +1,7 @@
 # Container Builder
 
-This is a proof of concept for using [Kaniko](https://github.com/GoogleContainerTools/kaniko/blob/main/docs/tutorial.md) and [Camel-K Builder package](https://github.com/apache/camel-k/tree/main/pkg/builder) to build Kogito services in a Kubernetes clusters.
+This project is now incorporated by the [Kogito Serverless Operator](). See the `container-builder` package. You can still use it in your projects since it's still a separated go module:
```
Comment (spolti) [inline README.md]: add the operator link?

File: README.md [hunk 2]
```diff
@@ -1,151 +1,7 @@
...
-`[[registry]]
-insecure = true
-location = "localhost:5000"`
-
-
-NOTE on Registry container
-TO enable the images deletion you need to set the following environment variable:
-```bash
-REGISTRY_STORAGE_DELETE_ENABLED=true
+```go
```
Comment (spolti) [inline README.md:5]: maybe can be removed as well.

Comment (ricardozanini) [reply README.md:5]: Why? This is to add color to the code snippet lol

Comment (spolti) [reply README.md:5]: I meant, remove everything.


if you decide to add the in the description above..

Sry if I was not clear. =/

Comment (ricardozanini) [reply README.md:5]: I'll add the description and keep the project in read-only mode. We don't need to remove everything.

**Review by spolti — APPROVED** [review]

**Review by radtriste — APPROVED** [review]

**Review by davidesalerno — APPROVED** [review]

**Review by desmax74 — APPROVED** [review]

**Pull request merged**